lib/upgrade/etcd/conversionv1v3/names.go

```diff
@@ -66,6 +66,11 @@ func convertNameNoDots(v1Name string) string {
 	return name
 }
 
+// Currently just an alias for convertName but it is expected to be different
```

Comment:
I think we probably don't want to do this aliasing - once we have a function we'll go around and find all the places we need to update.

For now, let's just use `convertName` in line.